sdk/keyvault/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,122 @@
+# Troubleshooting Azure Key Vault SDK Issues
+
+The Azure Key Vault SDKs for JavaScript use a common HTTP pipeline and authentication to create, update, and delete secrets,
+keys, and certificates in Key Vault and Managed HSM. This troubleshooting guide contains steps for diagnosing issues
+common to these SDKs.
+
+For any package-specific troubleshooting guides, see any of the following:
+
+* [Troubleshooting Azure Key Vault Administration SDK Issues](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/keyvault/keyvault-admin/TROUBLESHOOTING.md)
+* [Troubleshooting Azure Key Vault Certificates SDK Issues](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/keyvault/keyvault-certificates/TROUBLESHOOTING.md)
+* [Troubleshooting Azure Key Vault Keys SDK Issues](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/keyvault/keyvault-keys/TROUBLESHOOTING.md)
+* [Troubleshooting Azure Key Vault Secrets SDK Issues](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/keyvault/keyvault-secrets/TROUBLESHOOTING.md)
+
+## Table of Contents
+
+* [Troubleshooting Authentication Issues](#troubleshooting-authentication-issues)
+  * [HTTP 401 Errors](#http-401-errors)
+    * [Frequent HTTP 401 Errors in Logs](#frequent-http-401-errors-in-logs)
+    * [AKV10032: Invalid issuer](#akv10032-invalid-issuer)
+    * [Other Authentication Issues](#other-authentication-issues)
+  * [HTTP 403 Errors](#http-403-errors)
+    * [Operation Not Permitted](#operation-not-permitted)
+* [Other Service Errors](#other-service-errors)
+  * [HTTP 429: Too Many Request](#http-429-too-many-requests)
+
+## Troubleshooting Authentication Issues
+
+### HTTP 401 Errors
+
+HTTP 401 errors may indicate problems authenticating.
+
+#### Frequent HTTP 401 Errors in Logs
+
+Most often, this is expected. Azure Key Vault issues a challenge for initial requests that force authentication. You may see these errors most often during application startup, but may also see these periodically during the application's lifetime when authentication tokens are near expiration.
+
+If you are not seeing subsequent exceptions from the Key Vault SDKs, authentication challenges are likely the cause.
+
+#### AKV10032: Invalid issuer
+
+You may see an error similar to:
+
+```text
+
```

Comment:
I'd get rid of the extraneous blank lines before and after the block of code as well. Markdown is very sensitive to blank lines.